closed/src/java.base/share/classes/com/sun/crypto/provider/NativeGaloisCounterMode.java

```diff
@@ -326,40 +328,51 @@ int encrypt(byte[] in, int inOfs, int len, byte[] out, int outOfs) {
      */
     int encryptFinal(byte[] in, int inOfs, int len, byte[] out, int outOfs)
         throws IllegalBlockSizeException, ShortBufferException {
+        byte[] keyCopy;
```

Comment:
The first three don't need to be declared out of the scope where they are used.

Reply:
fixed